README.md

```diff
@@ -59,17 +59,29 @@ $vEvent
 $vCalendar->addComponent($vEvent);
 ```
 
-#### 5. Set HTTP-headers
+#### 5. Genreate a unique filename
 
 ```PHP
-header('Content-Type: text/calendar; charset=utf-8');
-header('Content-Disposition: attachment; filename="cal.ics"');
+$fileName = sha1(time() . time());
```

Comment:
Actually this will probably add the same number twice. It is very likely that `time()` returns the same number. To generate a unique filename, a function that generates a UUID is probably better. Furthermore, why do we need a unique filename here? Normally you will pick the filename by something meaningful? Especially, if the iCal is subscribed, the name does not matter?